README.md

```diff
@@ -45,13 +45,17 @@ You also need redis running locally. Please visit [redis][redis-url] for more in
 
 **String Uri**
 ```js
+var Seneca = require("seneca")
+var seneca = Seneca()
```

Comment:
var seneca = require("seneca")()
the 2 lines can be replaced with the above line